src/Yaapii.Atoms/IO/WriterTo.cs

```diff
@@ -93,7 +93,7 @@ private WriterTo(Func<StreamWriter> fnc) : this(new ScalarOf<StreamWriter>(fnc))
         /// <param name="tgt">the target streamwriter</param>
         private WriterTo(IScalar<StreamWriter> tgt) : base(new DeadStream())
         {
-            this._target = new StickyScalar<StreamWriter>(tgt);
+            this._target = new StickyScalar<StreamWriter>(tgt, writer => !writer.BaseStream.CanWrite);
```

Comment:
Do I get your idea correctly: The writer writes, until its stream is closed (maybe from a using on the outside). Then, the stream can restart itself if its needed (when someone wants to keep on writing).

Its a good idea, but I have some concerns about us taking the decision whether the stream is writable or not away from the user. Maybe someone wants the stream to stay closed?